media-types.md

```diff
@@ -3,6 +3,7 @@
 The following `mediaType` MIME types are used by the formats described here, and the resources they reference:
 
 - `application/vnd.oci.descriptor.v1+json`: [Content Descriptor](descriptor.md)
+- `application/vnd.oci.descriptor.foreign.v1+json`: [Content Descriptor](descriptor.md)
```

Comment:
There is no `application/vnd.oci.descriptor.foreign.v1+json`. That doesn't exist.